R/date_standardization_helpers.R

```diff
@@ -1,9 +1,11 @@
 #' Check date time frame
 #'
 #' @param first_date A Date object specifying the first valid date.
-#'    The arbitrary default value is fifty years before the `last_date`.
+#'    The arbitrary default value is fifty years before the \code{last_date}.
+#'    This can also be a character in ISO8601 format i.e. "2024-12-31".
 #' @param last_date A Date object specifying the last valid date.
```

Comment:
What other datatype can it be? Here you are indicating that it can also be character without saying what it usually is.

Reply:
I have mentioned on line `3` that it's supposed to be a `<Date>` object, the second option being `<character>`.